rails_docker/Gemfile.txt

```diff
@@ -73,6 +73,7 @@ group :test do
   gem 'simplecov', require: false # code coverage analysis tool for Ruby
   gem 'vcr' # Gem for recording test suite's HTTP interactions
   gem 'timecop' # Gem for time travel
+  gem 'public_suffix'
```

Comment:
What is the gem for? As you can see all gems have a short description to document what it's used for. So you always need to add such comment. In addition, if it's not related to the task at hand, prefer opening a separate PR for this change.

Reply:
will remove, not necessary anymore.